bar.c

```diff
@@ -1147,7 +1158,7 @@ main (int argc, char **argv)
                         case XCB_BUTTON_PRESS:
                             press_ev = (xcb_button_press_event_t *)ev;
                             {
-                                area_t *area = area_get(press_ev->event, press_ev->event_x);
+                                area_t *area = area_get(press_ev->event, press_ev->detail, press_ev->event_x);
                                 /* Respond to the click */
                                 if (area && area->button == press_ev->detail) {
```

Comment:
`area->button` is already checked in the new `area_get` function, see [here](https://github.com/otommod/bar/blob/cece20b562adf3d30c2f2a071736e82371559472/bar.c#L230). Should this check be kept here?